frontend/javascripts/oxalis/model/reducers/skeletontracing_reducer_helpers.ts

```diff
@@ -859,3 +859,25 @@ export function removeMissingGroupsFromTrees(
   });
   return changedTrees;
 }
+export function extractPathAsNewTree(
+  state: OxalisState,
+  sourceTree: Tree,
+  pathOfNodeIds: number[],
+): Maybe<Tree> {
+  return createTree(state, Date.now()).map((newTree) => {
```

Comment:
```suggestion
  return createTree(state, Date.now(), true, `Shortest path from node ${pathOfNodeIds[0]} to ${...}`).map((newTree) => {
```

the `true` is so that the tree is added to the active group. the fourth parameter doesn't exist yet, but it would be cool if you could add it as an optional name parameter. Then, the tree would get a nice descriptive name :)